test/k4FWCoreTest/src/components/ExampleFunctionalTransformerRuntimeCollectionsMultiple.cpp

```diff
@@ -180,8 +192,50 @@ struct ExampleFunctionalTransformerRuntimeCollectionsMultiple final
       trackVecOut.emplace_back(std::move(coll));
     }
 
+    if (recoVec.size() != 3) {
+      throw std::runtime_error("Wrong size of the reco vector, expected 3, got " + std::to_string(recoVecOut.size()) +
+                               "");
+    }
+    for (auto& recoColl : recoVec) {
+      auto coll = edm4hep::ReconstructedParticleCollection();
+      // if (recoColl->size() != 5 || recoColl->at(1).getPDG() != 1) {
+      //   std::stringstream error;
+      //   // error << "Wrong data in reco collection, expected 0, 1, 2, 3, 4 got " << coll[0].getPDG() << ", "
+      //   //       << coll[1].getPDG() << ", " << coll[2].getPDG() << ", " << coll[3].getPDG() << ", " << coll[4].getPDG();
+      //   throw std::runtime_error(error.str());
+      // }
```

Comment:
Is this a left over from developing? I think if this passes for all the tests we could just uncomment it and leave it in. Similar below.

Reply:
They are uncommented now